src/SQLite.cs

```diff
@@ -2562,54 +2595,77 @@ public class Column
 
 			public bool StoreAsText { get; private set; }
 
-			public Column (PropertyInfo prop, CreateFlags createFlags = CreateFlags.None)
+			public Column (MemberInfo member, CreateFlags createFlags = CreateFlags.None)
```

Comment:
Again I don't want to break public API, so please add the `PropertyInfo` version as an overload.

Reply:
Added below